SparkPost/2web2ui
# SA-1866: Break Down By report missing targeted and opens data

[SA-1866](https://sparkpost.atlassian.net/browse/SA-1866)

### What Changed
 - Changed `<ReportTable />` to use ReportBuilder metrics provided by `useReportBuilderContext` 

### How To Test

- Verify the bug was fixed using the steps to reproduce from the ticket
- Verify the summary report aggregate table does not break because redux state is being shared between both.
- Verify "Save as CSV" button works
